src/components/interaction-designer/toolbar/ErrorNotifications.vue

```diff
@@ -107,14 +107,7 @@ export default class ErrorNotifications extends mixins(Routes, Lang) {
   }
 
   get blockValidationStatuses(): { [key: string]: IValidationStatus } {
-    const blocksMap = this.activeFlow?.blocks.reduce((map: { [key: string]: boolean }, block) => {
```

Comment:
When thinking about this block of code, I noticed we can remove the mapping because block's validation are already prefixed with `block/`

Reply:
@jcbashdown , nice catch here, I think you're right !
I reverted by changes and added a doc string to the getter to make sure we understand that mapping part.